Code/Engine/GameEngine/GameApplication/Implementation/GameApplicationBase.cpp

```diff
@@ -405,6 +405,12 @@ ezApplication::ApplicationExecution ezGameApplicationBase::Run()
 
   Run_FinishFrame();
 
+  // continuously unload resources that are not in use anymore
+  ezResourceManager::FreeUnusedResources(ezTime::Microseconds(100), ezTime::Seconds(10.0f));
```

Comment:
NO: Run_FinishFrame will call present as its very last step, after that the application must update its clock asap. There should be nothing in between especially not things that have variable timing, otherwise the global clock and the framecounter is very jittery.

Reply:
Better ?